crates/tauri-plugin-holochain-service-consumer/android/src/main/java/HolochainServiceConsumerPlugin.kt

```diff
@@ -75,8 +83,30 @@ class HolochainServiceConsumerPlugin(private val activity: Activity): Plugin(act
         Log.d(TAG, "signZomeCall")
         val args = invoke.parseArgs(ZomeCallUnsignedFfiInvokeArg::class.java)
         serviceScope.launch(Dispatchers.IO) {
-            val res = serviceClient.signZomeCall(args.toFfi())
-            invoke.resolve(JSObject(res.toJSONObjectString()))
+            try {
+                val res = serviceClient.signZomeCall(args.toFfi())
+                invoke.resolve(JSObject(res.toJSONObjectString()))
+            } catch (e: Exception) {
+                handleCommandException(e, invoke)
+            }
+        }
+    }
+
+    /**
+     * Display the service notice if the exception is HolochainServiceNotConnectedException
+     */
+    private fun handleCommandException(e: Exception, invoke: Invoke) {
```

Comment:
If the exception is a HolochainServiceNotConnectedException, display the DisconnectNotice once the webview is initialized.

If we attempt to display the notice before the webview is initialized it doesn't work. I didn't dig into why. My assumption is that the webview is the root view of the application, so without it we have no view to attach the notice to.